js/tweek-client/src/utils.ts

```diff
@@ -2,6 +2,19 @@ export function captialize(string) {
   return string.charAt(0).toUpperCase() + string.slice(1);
 }
 
+export const fetchWithTimeout = (timeout, fetch): Promise<Response> => {
```

Comment:
I thought something more like this:
```
const createFetchWithTimeout = (timeout, fetch) => (input, init) => {
  return Promise.race([fetch(input, init), requestTimeout(timeout)]);
}
```
and then use it like this:
```
fetchClient = createFetchWithTimeout(requestTimeoutInMillis, fetchClient )
```

Reply:
Well, I think this way "fetch" is more abstract, I just wrap it in a Race with timeout...I don't care what params you give it..

Reply:
Anyways I need this change to be merged for duty issues...So we can do the 2nd interation later

Reply:
Fetch is a standard spec, multiple environments are implementing it like node, browser vendors and RN native. That's the reason we are using it. So it's ok to expect there parameters

Reply:
I don't think 5 more minutes will make that big of a difference. We can make these changes before merging